vm/devices/net/netvsp/src/lib.rs

```diff
@@ -460,6 +472,7 @@ struct QueueStats {
     tx_checksum_packets: Counter,
     tx_packets_per_wake: Histogram<10>,
     rx_packets_per_wake: Histogram<10>,
+    tx_packets_coalesced: SharedCounter,
```

Comment:
Actually, we don't need to track the coalesced packets in two places and sync. As long as tx_packets_coalesced is tracked in the ManaQueue, they should show up when inspected.